cmd/mt-whisper-importer-reader/main.go

```diff
@@ -137,6 +137,10 @@ func main() {
 
 	getFileListIntoChan(pos, fileChan)
 	wg.Wait()
+
+	processed := atomic.AddUint32(&processedCount, 1)
```

Comment:
why does that add `1` instead of just loading the value

Reply:
cause i hastily copy pasted this code.fixed :)